genkit-tools/cli/src/commands/ui-start.ts

```diff
@@ -126,14 +126,14 @@ async function startAndWaitUntilHealthy(
   serversDir: string
 ): Promise<ChildProcess> {
   return new Promise((resolve, reject) => {
-    const serverPath = path.join(__dirname, '../utils/server-harness.js');
     const child = spawn(
-      'node',
-      [serverPath, port.toString(), serversDir + '/devui.log'],
+      process.execPath,
```

Comment:
does this end up being the bundled bun binary?

Reply:
Oh, I see the other comment... I need to think about how I feel about it... probably fine....

Reply:
@apascal07

Reply:
I'm not referring the command itself, but the underlying web-server being run from the harness in a separate process.

Reply:
Ah, now I get it. Sorry, took me a sec to connect the dots. Yes, the refactoring is unblocked for sure.